.gitignore

```diff
@@ -1,7 +1,7 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
 .env
-
+.nx
```

Comment:
What's in `.nx`? Ideally we should have a comment explaing why it's ignored.

Reply:
done